lib/classes/organisations_user_builder.rb

```diff
@@ -15,6 +15,8 @@ def initialize(params)
     @user = @organisations_user.user if @organisations_user
     @organisation_id = params["organisation_id"].presence.try(:to_i)
     @user_attributes = params['user_attributes']
+    @user_address_attributes = params['user_address_attributes']
+    @district = District.find(@user_address_attributes['district_id'].presence&.to_i) rescue nil
```

Comment:
We can use find_by_id here and remove rescue nil.

Reply:
done already